### PR TITLE
perf(Logger): change base log fields

### PR DESCRIPTION
**Pull request by OlivierCuyp**
improve error serializing

### CODE REVIEW

File: src/libs/Logger.js [hunk 1]
```diff
@@ -46,19 +43,43 @@ module.exports = class Logger {
...
-    if (data instanceof Error) {
-      let error = data;
+  serializeError (logLevelindex, logData) {
+    let serialized;
+    const isError = logData instanceof Error;
+    let baseError;
+
+    if (isError) {
+      baseError = logData;
+    } else if ((logData || {}).error instanceof Error) {
```
Comment (SagaStalin) [inline src/libs/Logger.js]: MAJOR Add the missing "else" clause. [rule](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AElseIfWithoutElse)

File: src/libs/Logger.js [hunk 2]
```diff
@@ -46,19 +43,43 @@ module.exports = class Logger {
...
       if (logLevelindex < this.stackLevelIndex) {
         // Remove stack when not required
-        error = this.removeStack(error);
+        baseError = this.removeStack(baseError);
       }
-      data = {
-        error: pino.stdSerializers.err(error)
-      };
+      serialized = isError
+        ? { error: pino.stdSerializers.err(baseError) }
```
Comment (SagaStalin) [inline src/libs/Logger.js]: INFO Convert this usage of the ternary operator to an "if"/"else" structure. [rule](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AConditionalOperator)

**Comment by SagaStalin** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by OlivierCuyp** [comment]
[2(error-logging): Improve error logging](https://trello.com/c/z9DDJqij/3795-2error-logging-improve-error-logging)

**Pull request merged**